webcompat/static/js/lib/issues.js

```diff
@@ -482,11 +485,12 @@ issues.MainView = Backbone.View.extend({
       });
     }
   },
-  addNewComment: function() {
+  addNewComment: function(event) {
     var form = $(".js-Comment-form");
     var textarea = $(".js-Comment-text");
-    // Only bother if the textarea isn't empty
-    if ($.trim(textarea.val())) {
+
+    if (form[0].checkValidity()) {
```

Comment:
I guess this would throw in older browsers... but we probably don't support them anyways?

Reply:
What would older browsers mean here? :)

Reply:
I guess IE9 and Opera Mini would be the biggest, but I'm guessing we get exactly 0% visits from those browsers...